src/utils/cat_api.py

```diff
@@ -1,11 +1,11 @@
 import aiohttp
 
-CAT_API_URL: str = "https://api.thecatapi.com/v1/images/search"
+CAT_URL: str = "https://api.thecatapi.com/v1/images/search"
 
 
 async def get_cat_image_url() -> str:
-    """Get a URL for a random cat image."""
-    async with aiohttp.ClientSession() as client, client.get(CAT_API_URL) as response:
+    """Get an image of a random cat."""
+    async with aiohttp.ClientSession() as client, client.get(CAT_URL) as response:
```

Comment:
Why was this reverted? I changed the docstring and the variable name for a reason.

Then again, if the cats are being removed, this entire file can be removed entirely.

Reply:
I was explaining how to do rebase in vc and had no idea how interactive rebase worked in vscode and probably fucked up my instructions at some point. blame me

Reply:
ill change it

Reply:
Resolved in favor of my next code review, which mentions this alongside some other issues.